assets/parquet/main.py

```
@@ -0,0 +1,152 @@
+        if mode == 'json string':
+            self.properties['mode'] = 'json string'
+        elif mode == 'pairwise arrays':
+            self.properties['mode'] = 'pairwise arrays, no memory restriction)'
+
+    @property
+    def limits_param(self):
+        if self.pairwise_arrays:
+            return [
+                    '--cpus=4'
+                    ]
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)